packages/bits/src/lib/wizard-v2/wizard.component.less

```diff
@@ -34,31 +34,42 @@
     }
 }
 
+.nui-step-header-wrapper{
+    position: relative;
+}
+
 .nui-wizard-horizontal-line {
```

Comment:
All of the horizontal-specific styles should live in the wizard-horizontal less file, and all of the vertical-specific files should live in the wizard-vertical less file.